miniapps/nurbs/nurbs_solenoidal.cpp

```diff
@@ -0,0 +1,400 @@
+// Copyright (c) 2010-2024, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+//
+//        ------------------------------------------------------------
+//        NURBS Solenoidal Miniapp: Project solenoidal velocity
+//        ------------------------------------------------------------
+//
+//
+// Compile with: make nurbs_solenoidal
+//
+// Sample runs:  nurbs_solenoidal -m ../../data/square-nurbs.mesh -o 2
+//               nurbs_solenoidal -m ../../data/cube-nurbs.mesh -o 2
+//               nurbs_solenoidal -m ../../data/pipe-nurbs-2d.mesh -o 2
```

Comment:
The `pipe-nurbs-2d.mesh` sample run results in an error. I'm not sure if this intended, or if this will cause problems with testing.

Reply:
Multipath mesh therefore removed it.